### PR TITLE
Prepare to major release

### PR DESCRIPTION
**Pull request by ilfa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,13 +44,13 @@ If you'd like to have a similar SPA wrapper for the OSS version of FingerprintJS
...
 - [Getting Started](#getting-started)
 - [Support + Feedback](#support--feedback)
-- [What is Fingerprint Pro](#what-is-fingerprint-pro)
 - [License](#license)
 
 ## Documentation
 
 This library uses [Fingerprint Pro](https://fingerprint.com/github/) under the hood, you can view the document for the core technology.
-- [Documentation](https://dev.fingerprint.com/docs)
+- To learn more about the Fingerprint Pro read our [product documentation](https://dev.fingerprint.com/docs).
```
Comment (JuroUhlar) [inline README.md:52]:
```suggestion
- To learn more about Fingerprint Pro read our [product documentation](https://dev.fingerprint.com/docs).
```

File: README.md [hunk 2]
```diff
@@ -41,13 +44,13 @@ If you'd like to have a similar SPA wrapper for the OSS version of FingerprintJS
...
+- To learn more about this SDK, there is [Typedoc-generated SDK Reference](https://fingerprintjs.github.io/fingerprintjs-pro-spa) available.
```
Comment (JuroUhlar) [inline README.md:53]:
```suggestion
- To learn more about this SDK, there is a [Typedoc-generated SDK Reference](https://fingerprintjs.github.io/fingerprintjs-pro-spa) available.
```

Comment (JuroUhlar) [reply README.md:53]: Also, is it just me or does https://fingerprintjs.github.io/fingerprintjs-pro-spa/ give you a 404 now?

**Review by JuroUhlar — CHANGES_REQUESTED** [review]

**Review by TheUnderScorer — APPROVED** [review]

**Comment by kdawgwilk** [comment]
What breaking changes were actually included in 1.0.0?  It says breaking changes but mentions no breaking changes that we need to be aware of?

**Comment by JuroUhlar** [comment]
Hm it seems the semantic versioning release process automatically puts a big scary BREAKING CHANGES title on any major version release, which makes sense most of the time but not when you go `0.x` to `1.0` without breaking anything, which I think is the case here. @ilfa There are no actual braking changes, correct?

**Pull request merged**